porcupine/plugins/text_google_search.py

```diff
@@ -0,0 +1,21 @@
+"""Search selected text on Google."""
+
+import tkinter as tk
+from porcupine import menubar, tabs
+import webbrowser
+
+def google_search(tab: tabs.FileTab) -> None:
+    try:
+        selected_text_tuple = tab.textwidget.tag_ranges(tk.SEL)
+        # Check if text is selected to search on google
+        if len(selected_text_tuple)>0:
+            selected_text = tab.textwidget.get(*selected_text_tuple)
```

Comment:
Slightly simpler:

```suggestion
            selected_text = tab.textwidget.get("sel.first", "sel.last")
```

Reply:
It has been simplified in commit https://github.com/Akuli/porcupine/pull/1024/commits/4ec19862ec918996f7091e86d98b887f01d5ecea